<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Morfologia</class>
 <widget class="QMainWindow" name="Morfologia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Morfologia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelOryginal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBinarny">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrzetworzony">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>521</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="Dylatacja">
     <property name="text">
      <string>Dylatacja</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Erozja">
     <property name="text">
      <string>Erozja</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Otwarcie">
     <property name="text">
      <string>Otwarcie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Domkniecie">
     <property name="text">
      <string>DomkniÄ™cie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TopHat">
     <property name="text">
      <string>Top hat</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BottomHat">
     <property name="text">
      <string>Bottom hat</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Obraz oryginalny</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>  Obraz binarny</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> Obraz przetworzony</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNapis">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOtworz"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOtworz">
   <property name="text">
    <string>Otworz</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
